<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConGUISettingsDialog</class>
 <widget class="QDialog" name="ConGUISettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Location of AgentGUI</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Locate AgentGUI:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="xmmeLocationLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="xmmeLocationButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Location of PopGUI</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Locate PopGUI:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="popguiLocationLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="popguiLocationButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Location of GXParser</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Locate GXParser:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gxparserLocationLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="gxparserLocationButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Location of ExpGUI</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Locate ExpGUI:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="simulatorLocationLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="simulatorLocationButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Location of VisGUI</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Locate VisGUI:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="visguiLocationLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="visguiLocationButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConGUISettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConGUISettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
